proxy/src/main/java/com/velocitypowered/proxy/connection/client/ClientPlaySessionHandler.java

```diff
@@ -229,7 +230,9 @@ public boolean handle(PluginMessage packet) {
         player.getKnownChannels().removeAll(PluginMessageUtil.getChannels(packet));
         backendConn.write(packet.retain());
       } else if (PluginMessageUtil.isMcBrand(packet)) {
-        player.setClientBrand(PluginMessageUtil.readBrandMessage(packet.content()));
+        String brand = PluginMessageUtil.readBrandMessage(packet.content());
+        server.getEventManager().fire(new PlayerClientBrandEvent(player, brand));
```

Comment:
```suggestion
        server.getEventManager().fireAndForget(new PlayerClientBrandEvent(player, brand));
```
I forgot about this, it seems.

Reply:
Makes sense to me -- it's done above for `PlayerChannelRegisterEvent` as well.